pkg/controller/workload/common/fake_helpers.go

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package common
+
+import (
+	"fmt"
+	"log"
+
+	"net/netip"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/rand"
+	"kmesh.net/kmesh/api/v2/workloadapi"
+)
+
+func CreateFakeService(name, ip, waypoint string, optional ...interface{}) *workloadapi.Service {
```

Comment:
I think this is not right, optional means you allow to pass in any types of values but the inner implement you only handle `*workloadapi.LoadBalancing`

Reply:
Have you ppushed your new commit?

Reply:
give it a check now @hzxuzhonghu